examples/draft-0-10-0/universal/readme.md

```diff
@@ -25,5 +25,3 @@ then, run
 `npm run demo`
 
 which will open a server listening on [http://localhost:3003](http://localhost:3003)
-
-visiting the main route will return a simple unstyled draft editor but the client, on re-rendering the `<SimpleEditor />` will complain that there is a mismatch between the server rendered content and the client-rendered content.
```

Comment:
is this no longer true?

Reply:
Right :) - @nsfmc and  @pofigizm fixed that by adding the `editorKey` prop, and this example showing how to use it.